lib/galaxy/datatypes/blast.py

```diff
@@ -0,0 +1,281 @@
+"""NCBI BLAST datatypes.
+
+Covers the ``blastxml`` format and the BLAST databases.
+"""
+
+import logging
+import os
+from time import sleep
+
+from galaxy.datatypes.data import get_file_peek
+from galaxy.datatypes.data import Data, Text
+from galaxy.datatypes.xml import GenericXml
+
+log = logging.getLogger(__name__)
+
+
+class BlastXml(GenericXml):
+    """NCBI Blast XML Output data"""
```

Comment:
@peterjc can we add:

```
edam_format = "format_3331"
```

Not sure about `edam_data = "data_1381"` (http://bioportal.bioontology.org/ontologies/EDAM/?p=classes&conceptid=http%3A%2F%2Fedamontology.org%2Fdata_1381)

It should look like this: https://github.com/galaxyproject/galaxy/blob/3fd0e405fd6cb3333397ec19e627a4cf6ae36fe6/lib/galaxy/datatypes/proteomics.py#L21